bundles/org.openhab.binding.haywardomnilogic/src/main/resources/OH-INF/thing/chlorinator.xml

```diff
@@ -51,7 +50,7 @@
 		<item-type>Number:Dimensionless</item-type>
 		<label>Salt Output (%)</label>
 		<description>Current salt output setting for the chlorinator (%).</description>
-		<state min="0" max="100" step="1.0" pattern="%d %unit%" readOnly="false"/>
+		<state min="0" max="100" step="1.0" pattern="%d %" readOnly="false"/>
```

Comment:
%% for the unit

Reply:
This does not seem to be solved yet

Reply:
I hadn't updated this yet.  My IDE wasn't working last night.

Reply:
Ah, sorry i missed this open comment and allready merged it. If you can create a follow up PR with this fix i can merge it really soon.

Reply:
For reference: https://github.com/openhab/openhab-addons/pull/16733